phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -958,10 +958,6 @@ private void addCoprocessors(byte[] tableName, HTableDescriptor descriptor, PTab
                             opts.put(NonTxIndexBuilder.CODEC_CLASS_NAME_KEY, PhoenixIndexCodec.class.getName());
                             IndexRegionObserver.enableIndexing(descriptor, PhoenixIndexBuilder.class, opts, priority);
                         }
-                        if (descriptor.hasCoprocessor(Indexer.class.getName())) {
```

Comment:
@kadirozde Looks like the addendum patch didn't make it to 4.x branches? removing the repeated lines in my patch.